src/main/resources/db/migration/V1__init_tables.sql

```diff
@@ -16,5 +16,19 @@ CREATE TABLE IF NOT EXISTS users
     password TEXT NOT NULL,
     first_name TEXT NOT NULL,
     last_name TEXT NOT NULL,
-    shipping_address TEXT
+    shipping_address TEXT,
+    is_deleted BOOLEAN DEFAULT FALSE
+);
+
+CREATE TABLE IF NOT EXISTS roles
+(
+    id BIGSERIAL PRIMARY KEY,
+    name TEXT NOT NULL UNIQUE
+);
+
+CREATE TABLE IF NOt EXISTS users_roles
+(
+    user_id BIGINT REFERENCES users(id),
+    role_id BIGINT REFERENCES roles(id),
+    PRIMARY KEY (user_id, role_id)
```

Comment:
Please stop editing patch-sets with addition of new logic.
You're starting a bad habit for yourself.
1) In real life you have project that is already working and you task in to maintain and develop it.
As a result you cannot drop db to add new table or alter existing one. But after patch-set is edited your flyway will refuse to accept db due to checksum mismatch.
2) Depending on db setup big patch-sets have more chances to fail. Better to have 20 small patch-sets than 2 huge ones
3) It's much easier to check when and what was done to db when you have new patch-set for every PR. Not always you have everything fine and no bugs. Some day you will need urgently check something due to critical bug and properly rollback a feature without loss of data in db

Reply:
yeah, makes sense

Reply:
can we proceed with this flyway versions? I'll create the new one in case db structure change

Reply:
do not change this file and add a new one